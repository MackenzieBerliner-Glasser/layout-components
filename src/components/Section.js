import Component from './Component';
import styles from './Section.css';

export default class Section extends Component {
  render() {
    const images = this.props.images.map(({ image }) => {
      return `<img src="${image}">`;
    });

    return `
      <section>
        <div class="${styles.container}">
          ${images.join(' ')}
        </div>
      </section>
    `;
  }
}
