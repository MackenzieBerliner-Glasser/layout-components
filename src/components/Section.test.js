import Section from './Section';


describe('Section Component', () => {
  it('is a section', () => {
    const images = [
      {
        image: 'http://en.bcdn.biz/Images/2018/6/6/6483b0a6-667e-4209-9b66-5b4bce6b4e53.jpg'
      }
    ];

    const section = new Section({ images });

    expect(section.render()).toMatchSnapshot();
  });
});
