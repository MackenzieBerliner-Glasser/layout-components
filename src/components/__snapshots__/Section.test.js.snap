// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Component is a section 1`] = `
"
      <section>
        <div class=\\"container\\">
          <img src=\\"http://en.bcdn.biz/Images/2018/6/6/6483b0a6-667e-4209-9b66-5b4bce6b4e53.jpg\\">
        </div>
      </section>
    "
`;
